analytics/src/main/java/com/segment/analytics/Analytics.java

```diff
@@ -318,6 +329,19 @@ public void onActivityStarted(Activity activity) {
           @Override
           public void onActivityResumed(Activity activity) {
             runOnMainThread(IntegrationOperation.onActivityResumed(activity));
+
+            if (shouldTrackApplicationLifecycleEvents
+                && numberOfActivities.incrementAndGet() == 1
+                && !isChangingActivityConfigurations.get()) {
+
+              Properties properties = new Properties();
+              if (firstLaunch.get()) {
+                properties.putValue("version", currentVersion).putValue("build", currentBuild);
+              }
+              properties.putValue("from_background", !firstLaunch.getAndSet(false));
```

Comment:
On first launch we want to ensure `from_background` is false, subsequent calls, this will be true.